dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -254,15 +258,14 @@ if [ "$RH_CHE_IS_V_FIVE" == "true" ]; then
                     yq ".\"data\".\"che-workspace-che-server-endpoint\" = \"https://che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io/wsmaster/api\" | 
                         .\"data\".\"che-host\" = \"che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io\" | 
                         .\"data\".\"infra-bootstrapper-binary-url\" = \"https://che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io/agent-binaries/linux_amd64/bootstrapper/bootstrapper\" | 
-                        .\"data\".\"che-api\" = \"https://che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io/api/\" | 
+                        .\"data\".\"che-api\" = \"https://che-$RH_CHE_PROJECT_NAMESPACE.dev.rdu2c.fabric8.io/api\" | 
```

Comment:
@ScrewTSW The `che-api` should not contain a trailing `/`. This makes the ws-agent fail at workspace start.

If you fix this in another PR, I'll change it here and will rebase my PR on your

Reply:
thank you for this find, I wouldn't have a single idea. :)